source/tool/chipsec/helper/linux/helper.py

```diff
@@ -79,6 +81,8 @@ class LinuxHelper(Helper):
 
     DEVICE_NAME = "/dev/chipsec"
     DEV_MEM = "/dev/mem"
+    MODULE_NAME = "chipsec"
+    GET_PAGE_IS_RAM = True
```

Comment:
Just to clarify on that comment. I was talking about GET_PAGE_IS_RAM. How would you see that GET_PAGE_IS_RAM flag being used?

Should we drop support for kernel older than 2.6.33 ?